server/libs/database/src/entities/admin-config.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Column, Entity } from 'typeorm';
+
+@Entity('admin_config')
+export class AdminConfigEntity {
+  @Column({ primary: true })
```

Comment:
Is there any difference between this and `@PrimaryColumn()`?

Reply:
Probably just a helper method for the same thing, can switch this too